### PR TITLE
fix: changing style of studentProfile

### PR DESCRIPTION
**Pull request by joaoprferreira**
#74

## Developer Checks

- [x] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [x] Tested on browser device
- [x] Tested on Mobile device

### CODE REVIEW

File: src/components/StudentProfile/styles.ts [hunk 1]
```diff
@@ -1,22 +1,40 @@
 import styled from 'styled-components';
 import { Typography } from '../Typography';
 
-export const Container = styled.section`
+export const Container = styled.div`
```
Comment (jonataspinto) [inline src/components/StudentProfile/styles.ts:4]: Acho que pode ser mantido como section.

**Review by jonataspinto — APPROVED** [review]

**Comment by jonataspinto** [comment]
Build quebrando @joaoprferreira

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.57 | ✅ |
| First Contentful Paint | 1.3 s |  |
| Time to Interactive | 4.9 s |  |
| Speed Index | 1.7 s |  |
| Total Blocking Time | 760 ms |  |
| Largest Contentful Paint | 6.7 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.93 | ✅ |
| First Contentful Paint | 2.0 s |  |
| Time to Interactive | 5.8 s |  |
| Speed Index | 2.4 s |  |
| Total Blocking Time | 110 ms |  |
| Largest Contentful Paint | 2.1 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by joaoprferreira** [comment]
> 

Fixed

**Comment by joaoprferreira** [comment]
> Build quebrando @joaoprferre

**Comment by joaoprferreira** [comment]
> Build quebrando @joaoprferreira

Fixed

**Comment by jonataspinto** [comment]
> > Build quebrando @joaoprferreira
> 
> Fixed

Merrrge

**Review by jonataspinto — APPROVED** [review]

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.73 | ✅ |
| First Contentful Paint | 1.3 s |  |
| Time to Interactive | 4.4 s |  |
| Speed Index | 1.5 s |  |
| Total Blocking Time | 280 ms |  |
| Largest Contentful Paint | 6.3 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.67 | ✅ |
| First Contentful Paint | 2.0 s |  |
| Time to Interactive | 5.7 s |  |
| Speed Index | 2.4 s |  |
| Total Blocking Time | 160 ms |  |
| Largest Contentful Paint | 11.8 s |  |
| Cumulative Layout Shift | 0.172 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Pull request merged**